src/api/helpers/map-CIS-countries/map-CIS-country/country-rating-is-a-or-b/index.test.ts

```diff
@@ -0,0 +1,40 @@
+import countryRatingIsAorB from '.';
+import { EXTERNAL_API_DEFINITIONS } from '../../../../constants';
+
+const {
+  CIS: { COUNTRY_RATINGS },
+} = EXTERNAL_API_DEFINITIONS;
+
+describe('helpers/map-CIS-countries/map-CIS-country/country-rating-is-a-or-b', () => {
+  describe.each(COUNTRY_RATINGS.A)('COUNTRY_RATINGS.A', (rating) => {
+    it(`should return true for ${rating}`, () => {
+      const result = countryRatingIsAorB(rating);
+
+      expect(result).toEqual(true);
+    });
+  });
+
+  describe.each(COUNTRY_RATINGS.B)('COUNTRY_RATINGS.B', (rating) => {
```

Comment:
Can a country have an A and B rating? if it can, can a test be added, ignore if not

Reply:
Nope, it can only have one rating